### ISSUE TITLE
Fails to install on Ubuntu 24.04

### ISSUE THREAD

**Issue by davidsneighbour** [keybase/client#26812]
Keybase is not installable via .deb file on Ubuntu 24.04 and the `apt install -f` method does not work.

```bash
❯ sudo dpkg -i Downloads/keybase_amd64.deb
[sudo] password for patrick:
Selecting previously unselected package keybase.
(Reading database ... 224676 files and directories currently installed.)
Preparing to unpack Downloads/keybase_amd64.deb ...
Unpacking keybase (6.2.8-20240306193933.e38523abbe) ...
dpkg: dependency problems prevent configuration of keybase:
 keybase depends on libappindicator1 | libayatana-appindicator1; however:
  Package libappindicator1 is not installed.
  Package libayatana-appindicator1 is not installed.
 keybase depends on libgconf-2-4; however:
  Package libgconf-2-4 is not installed.

dpkg: error processing package keybase (--install):
 dependency problems - leaving unconfigured
Processing triggers for gnome-menus (3.36.0-1.1ubuntu3) ...
Processing triggers for desktop-file-utils (0.27-2build1) ...
Processing triggers for hicolor-icon-theme (0.17-2) ...
Processing triggers for shared-mime-info (2.4-4) ...
Errors were encountered while processing:
 keybase

❯ sudo apt --fix-broken install
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Correcting dependencies... Done
The following packages will be REMOVED:
  keybase
0 upgraded, 0 newly installed, 1 to remove and 0 not upgraded.
1 not fully installed or removed.
After this operation, 456 MB disk space will be freed.
Do you want to continue? [Y/n]
```

`libappindicator` is installed, `libappindicator1` does not exist.

**Comment by readonlyuser1** [comment]
https://blueprints.launchpad.net/ubuntu/noble/+source/libappindicator/+changelog
![image](https://github.com/keybase/client/assets/89613988/66b40499-f9cd-424c-b9e5-6a0a0763991c)

**Comment by phouverneyuff** [comment]
Any updates on when it will be fixed?

**Comment by phouverneyuff** [comment]
```
cat /etc/os-release 
PRETTY_NAME="Ubuntu 24.04 LTS"
NAME="Ubuntu"
VERSION_ID="24.04"
VERSION="24.04 LTS (Noble Numbat)"
VERSION_CODENAME=noble
ID=ubuntu
ID_LIKE=debian
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
UBUNTU_CODENAME=noble
LOGO=ubuntu-logo
```


```
uname -r
6.8.0-31-generic
```

```
sudo apt-get upgrade    
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
You might want to run 'apt --fix-broken install' to correct these.
The following packages have unmet dependencies:
 keybase : Depends: libappindicator1 but it is not installable or
                    libayatana-appindicator1 but it is not installable
           Depends: libgconf-2-4 but it is not installable
E: Unmet dependencies. Try 'apt --fix-broken install' with no packages (or specify a solution).
```

**Comment by phouverneyuff** [comment]
fixed by [#26525](https://github.com/keybase/client/pull/26525)

**Comment by phouverneyuff** [comment]
How do I temporally solved it:

```
git clone git@github.com:vorlock/KBclient.git
cd KBClient/packaging/linux/
./build_binaries.sh prerelease # You will see a folder created after that and use this path on the last command. You need golang, npm and yarn installed
cd deb
./package_binaries.sh FOLDER_CREATED_AT_STEP_3
sudo apt install FOLDER_CREATED_AT_STEP_3/deb/amd/keybase***.deb
```

**Comment by davidsneighbour** [comment]
Be careful with unknown repositories. It does not seem that the keybase team is updating the issue at all. Using any other repo might result in unwanted results.

**Comment by phouverneyuff** [comment]
https://github.com/keybase/client/pull/26525/files

You can track all of the changes in this repo using the link above. 
This repo that I showed on the clone is the same one with PR. To feel safer, just use that hash-commit for that PR.

DYOR

**Comment by phouverneyuff** [comment]
![image](https://github.com/keybase/client/assets/9394724/89f2c8c7-a351-4cb8-ad54-0107a88ece7f)

**Comment by jurjendevries** [comment]
> How I temporally solved it:
> 
> ```
> git clone git@github.com:vorlock/KBclient.git

Hi @phouverneyuff , thanks for providing a solution. However, I am getting an error:

$ git clone git@github.com:vorlock/KBclient.git
Cloning into 'KBclient'...
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

**Comment by jurjendevries** [comment]
@phouverneyuff  sorry never mind, I just saw your link to https://github.com/keybase/client/pull/26525 and see that it has been fixed in a new version 6.3.1 :-) Thanks for your contribution to this.

**Comment by davidsneighbour** [comment]
The issues are indeed resolved by installing the latest release: https://keybase.io/docs/the_app/install_linux
